HQSmokeTests/testPages/home/home_page.py

```diff
@@ -26,6 +27,7 @@ def __init__(self, driver):
         self.alert_button_accept = (By.ID, "hs-eu-confirmation-button")
         self.application_path = (By.LINK_TEXT, UserData.village_application)
         self.mobile_workers_menu_link_text = (By.LINK_TEXT, "Mobile Workers")
+        self.show_full_menu_id = (By.ID, "commcare-menu-toggle")
```

Comment:
Just a question out of interest: where do you get these IDs? I know that is their actual element ID's, but do you have a record/register which keeps track of these elements' IDs? If not, I presume when this ID (or any other element's ID) changes for some reason, you will pick this up only when the test(s) fails?

Reply:
I got the IDs from the HTML element's attributes (using the console). We can also locate elements by [creating XPaths](https://www.guru99.com/xpath-selenium.html), but ID is the easiest to fetch and changes the least.

As for keeping a record/register, no, we don't, and yes, the tests will fail if anything changes in the HTML since selenium works around the HTL elements, we try to use XPaths that are least likely to change.